cloudomate/test/test_captchasolver.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+import requests
+import json
+import os
+import base64
+import time
+
+from cloudomate.util.captchasolver import CaptchaSolver, ReCaptchaSolver
+from mock import patch, MagicMock
+
+
+class TestCaptchaSolver(unittest.TestCase):
+
+    def setUp(self):
+        self.captcha_solver = CaptchaSolver("213389asd8u912823")
+        self.recaptcha_solver = ReCaptchaSolver("213389asd8u912824")
+
+    @patch("json.loads")
+    @patch("requests.post")
+    def test_get_balance(self, mock_requests, mock_json):
+        self.captcha_solver.get_balance()  
+
+
+          
+    def test_get_current_key(self):
+        self.assertEqual(self.captcha_solver.get_current_key(),"213389asd8u912823")
+        self.assertEqual(self.recaptcha_solver.get_current_key(),"213389asd8u912824")
+
+    @patch('time.sleep')
+    @patch('base64.b64encode')
+    @patch('builtins.open')
+    @patch('os.path.isfile')
+    def test_solve_captcha_text_case_sensitive(self, mock_os, mock_builtins, 
+                                               mock_base64, mock_time):
+        self.captcha_solver._create_task_captcha_text_case_sensitive = MagicMock()
+        self.captcha_solver._get_task_status = MagicMock()
+        self.captcha_solver._get_task_result = MagicMock()
+         
+        self.captcha_solver.solve_captcha_text_case_sensitive('test')
+
+        assert self.captcha_solver._create_task_captcha_text_case_sensitive.called
```

Comment:
Please use `self.assertTrue` (this method gives some more information compared to `assert`).